modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1057,8 +1091,17 @@ def _read(cls, filepath_or_buffer, **kwargs):
                 newline=newline,
                 header_size=header_size,
                 pre_reading=pre_reading,
+                read_callback_kw=None
+                if compute_metadata_before_skipping_rows
+                else read_callback_kw,
```

Comment:
[this comment](https://github.com/modin-project/modin/pull/5482#discussion_r1059023537) is not resolved, is it?

Reply:
this should be resolved now